packages/archetype/src/__tests__/__snapshots__/Archetype.spec.js.snap

```diff
@@ -125,19 +125,145 @@ exports[`Archetype Default 1`] = `
       }
     }
   >
-    <Example>
-      <div>
-        <Archetype>
-          <glamorous(div)>
-            <div
-              className="css-nil"
-            >
-              Archetype
-            </div>
-          </glamorous(div)>
-        </Archetype>
+    <LiveProvider
```

Comment:
It's not awesome that all of our snapshots will no longer match if we change our LivePreview styles. Wonder if there's a way around that.